src/remotedevDevtools.js

```diff
@@ -0,0 +1,171 @@
+// code taken from
```

Comment:
1. `remotedevDevtools` is a confusing name (doesn't mention Redux) and the pattern of capitalization is confusing.
2. It's not clear to me what this file is for... do we need all of it? And what for? Would it be better to just get it from the `remotedev` package? How much really needs to be customized?

Reply:
ad 1. I see, please note that `devTools.js` doesn't mention Redux either... I do not mind changing the name. Capitalization could be better(probably the uppercase `T`, like: `remotedevDevTools.js`). My thinking was: both files are originally called `devTools.js` so I wanted to put the name of the lib that it comes from as a prefix. If you have any better name for this file on your mind, please, let me know and I'm alright with changing it!


Quick research on whether it is worth putting up remotedeDevtools.js in `redux-remote-devtools` patch:
- devTools.js - patches `__REDUX_DEVTOOLS_EXTENSION_COMPOSE__`
- remotedevDevtools.js - patches `__REDUX_DEVTOOLS_EXTENSION__`
- this file contains a little bit of a different logic than devTools.js yet it's not that different from what I saw. I think we should be able to replace `__REDUX_DEVTOOLS_EXTENSION__` with `__REDUX_DEVTOOLS_EXTENSION_COMPOSE__`, BUT we have to consider how `__REDUX_DEVTOOLS_EXTENSION__` is called:
    - for `@react-navigation/devtools` instance(`root.react.js`) - unfortunately here it is called inside of the code of navigation. So, to change that, we'd need to post a patch for the navigation devtools package.
    - for the `SquadCal` instance(`dev-tools.react.js`) - this one is done directly in our code, so that'd be easier.
- Having all that said, I think pasting the code of `remotedev` as it is now is a good solution - the API matches(with that subscribe logic that's there) and it's not a lot of code.
- I checked how it's done in `react-native-debugger` and it seems that they take the default export of `devTools.js` and assing the following functions to it: `send`, `connect`, `disconnect`(check [here](https://github.com/jhen0409/react-native-debugger/blob/639c02c0f3385c83e62b0ca483cb989db0e4b1a4/app/worker/index.js#L25)). I'm not sure why: only the `connect` function is used, so the way I implemented it(patching the runtime with an object containing only the `connect` field which is a function) omits unnecessary code and still works the same.
- BTW here is a PR with the changes that have been applied here: https://github.com/karol-bisztyga/remotedev/pull/1/files

Reply:
1. Regarding the name, how about naming this file `connectToReduxDevServer.js`? Not sure, that might be too wordy...
2. Thanks for explaining this. I now understand why we have this and why it's necessary. I don't think we should replace `__REDUX_DEVTOOLS_EXTENSION__` with `__REDUX_DEVTOOLS_EXTENSION_COMPOSE__`
3. I think it would make more sense to include the `remotedev` package with a separate patchfile. That would we keep our patchfiles minimal, and makes it easier to get upstream changes that may come in future `remotedev` updates. (If we do this, ignore my comment above about the name)